app/upgrades/v1/constants.go

```diff
@@ -0,0 +1,20 @@
+package v1
+
+const UpgradeName string = "upgrade_v1"
```

Comment:
We are now in stage of `beta`. (In fact, the latest release version name is `v1.0.0-beta.1`.
And this `upgradeName` will be a `proposal name`.
So simpler one is better.
Let's switch to `v1-beta.2`.